devito/arch/compiler.py

```diff
@@ -408,11 +410,15 @@ def __init__(self, *args, **kwargs):
                                  '-fopenmp-targets=amdgcn-amd-amdhsa',
                                  '-Xopenmp-target=amdgcn-amd-amdhsa']
                 self.ldflags += ['-march=%s' % platform.march]
-        elif platform is M1:
+        elif platform is M1 and language == 'openmp':
```

Comment:
shouldn't this `and language == 'openmp'` be moved down in `and` with `if llvmm1` ?

Reply:
Not sure it matters much since we in ClangCompiler in practice but would be more logical yeah will fix

Reply:
fixed